packages/adblocker/src/filters/network.ts

```diff
@@ -1027,6 +1040,10 @@ export default class NetworkFilter implements IFilter {
       // --------------------------------------------------------------------- //
     }
 
+    if (isCaseSensitive === false) {
+      line = line.toLowerCase();
```

Comment:
this lowercases the options which wasn't the case previously. Are we sure this is safe? (I don't recall any option that is case sensitive)